README.md

```diff
@@ -1 +1,185 @@
-# Hello_Microverse
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+
+  
+  <br/>
+
+  <h3><b>Microverse README Template</b></h3>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+  - CSS
+  - Github workflow
+  - Linters
+  - HTML
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+
+# 📖 [Hello-Microverse] <a name="about-project"></a>
+
+**[Hello-Microverse]** is a...
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="#">HTML</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="#">CSS</a></li>
+  </ul>
+</details>
+
+
+### Key Features <a name="key-features"></a>
+
+- **[Home page]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+To get a local copy up and running, follow these steps.
+
+
+### Prerequisites
+
+In order to run this project you need:
+ - Vs code
+ - git
+
+
+### Setup
+
+Clone this repository to your desired folder:
+If you want to clone this repository, clone into your desired folder and enter "https://github.com/Blessman-Newton/Hello_Microverse-.git into your terminal
+
+### Install
+
+linters
+Html
+
+### Usage
+
+To run the project, execute the following command:
+- Live Server
+
+Open index.html in your browser
+### Run tests
+
+To run tests, run the following command:
+
+- npx stylelint "**/*.{css,scss}
+- hint.x
+
+### Deployment
+
+You can deploy this project using by transfering files in your local repository onto your saver.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 👥 Authors <a name="**Blessman-Newton">Blessman Newton</a>
+
+> Mention all of the collaborators of this project.
+
+👤 **Author1**
+
+- GitHub: [Blessman-Newton](https://github.com/Blessman-Newton)
+- Twitter: [@Blessman-Newton](https://twitter.com/blessmannewton)
+- LinkedIn: [Blessman_newton](https://linkedin.com/in/Blessmannewton)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> Describe 1 - 3 features you will add to the project.
+
+- [ ] **[new_feature_1]**
+- [ ] **[new_feature_2]**
+- [ ] **[new_feature_3]**
```

Comment:
- [ ] [OPTIONAL] Please make sure to provide at least 1 feature you'll be adding in the future as this shows you have a growth mindset which is one of the key traits of a professional developer.
    -  So please remove the above placeholder texts and write at least 1 feature.